lib/connection.js

```diff
@@ -21,6 +21,8 @@ var Connection = function (config) {
   config = config || {}
   this.stream = config.stream || new net.Socket()
   this._keepAlive = config.keepAlive
+  this._keepAliveIdleMillis = config.keepAliveIdleMillis
```

Comment:
wonder if we should call this `keepAliveInitialDelayMillis` to better indicate what it actually does? https://nodejs.org/api/net.html#net_socket_setkeepalive_enable_initialdelay

Reply:
I could not find a way to delay _accepting_ a new TCP connection in node.js. If the socket is not listening, then the OS will refuse the connection immediately. I have an other idea that I will have to test: creating the server in an other process, and blocking the event loop with busy waiting.